tests/integration/raw.rs

```diff
@@ -0,0 +1,82 @@
+// Copyright 2019 TiKV Project Authors. Licensed under Apache-2.0.
+
+use crate::{arb_batch, integration::pd_addr};
+use futures::executor::block_on;
+use proptest::{arbitrary::any, proptest};
+use tikv_client::{raw::Client, Config, KvPair, Value};
+
+proptest! {
```

Comment:
Could you add some comments to describe what each of these tests is testing please? (In particular for contributors who are not familiar with proptest).